modules/apps/object/object-api/src/main/java/com/liferay/object/definition/notification/term/ObjectDefinitionNotificationCurrentUserTerm.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.definition.notification.term;
+
+import com.liferay.portal.kernel.util.SetUtil;
+
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * @author Paulo Albuquerque
+ */
+public enum ObjectDefinitionNotificationCurrentUserTerm {
+
+	CURRENT_USER_EMAIL("current-user-email-address", "[%CURRENT_USER_EMAIL%]"),
+	CURRENT_USER_FIRST_NAME(
+		"current-user-first-name", "[%CURRENT_USER_FIRSTNAME%]"),
+	CURRENT_USER_ID("current-user-id", "[%CURRENT_USER_ID%]"),
+	CURRENT_USER_LAST_NAME(
+		"current-user-last-name", "[%CURRENT_USER_LASTNAME%]"),
+	CURRENT_USER_MIDDLE_NAME(
+		"current-user-middle-name", "[%CURRENT_USER_MIDDLENAME%]"),
+	CURRENT_USER_PREFIX("current-user-prefix", "[%CURRENT_USER_PREFIX%]"),
+	CURRENT_USER_SUFFIX("current-user-suffix", "[%CURRENT_USER_SUFFIX%]");
+
+	public static Set<String> getTermsName() {
+		return Collections.unmodifiableSet(
```

Comment:
@p-albuquerque 

I believe we can use `values()` instead of predefined values or even you can just create a method here in the enum to check if the term exists, [see](https://github.com/liferay/liferay-portal/blob/3a90bef9c0610676bd93cd66e9eb70fd816dfef4/modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-client/src/main/java/com/liferay/headless/commerce/admin/catalog/client/dto/v1_0/MappedProduct.java#L333-L343)

Reply:
good idea!! thanks